.github/workflows/python-publish.yml

```diff
@@ -14,6 +14,24 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        persist-credentials: false
+        fetch-depth: 0
+    - name: Set version veriable
+      id: vars
+      run: echo ::set-env name=RELEASE_VERSION::${GITHUB_REF#refs/*/}
+    - name: Update versiona rel
+      run: sed -i -E "s/version=\\\"[0-9.]*\\\",/version=\\\"$RELEASE_VERSION\\\",/g" ./setup.py
+    - name: Commit the changes
+      run: |
+      git add ./setup.py
+      git config --local user.email "action@github.com"
+      git config --local user.name "GitHub Action"
+      git commit -m "ci: Automatic version bump" -a | exit 0
+    - name: Push the changes
+      uses: ad-m/github-push-action@master
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why does it have to commit and push back?

Reply:
Answered here : https://github.com/paypay/paypayopa-sdk-python/pull/29#issuecomment-705312974